pkg/shell/indexes.js

```diff
@@ -519,7 +519,8 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
 
         if (!menu.length && !tools.length) {
             const group = document.createElement("li");
-            group.className = "list-group-item disabled";
+            group.className = "list-group-item";
+            group.setAttribute("disabled", "");
```

Comment:
this backfires a bit.
Current master:
![noresultsmaster](https://user-images.githubusercontent.com/12330670/74927046-375ae800-53d7-11ea-8c88-9d553f3c82f7.png)
This PR:
![noresultsthispr](https://user-images.githubusercontent.com/12330670/74927054-3b870580-53d7-11ea-9bff-6634c6dc566d.png)

The css is based on `.disabled` class, so it needs to be adjusted.

Reply:
Same case, this is also a li element, disabled should not apply to it. Which is my fault, oops.